patterns/cell-based-eks/1.cell1/README.md

```diff
@@ -0,0 +1,181 @@
+# Cell-Based Architecture for Amazon EKS
+
+This example shows how to provision a cell based Amazon EKS cluster.
+
+* Deploy EKS Cluster with one managed node group in a VPC and AZ
+* Deploy Fargate profiles to run `coredns`, `aws-load-balancer-controller`, and `karpenter` addons
+* Deploy Karpenter `Provisioner` and `AWSNodeTemplate` resources and configure them to run in AZ1
+* Deploy sample deployment `inflate` with 0 replicas
+
+Refer to the [AWS Solution Guidance](https://aws.amazon.com/solutions/guidance/cell-based-architecture-for-amazon-eks/) for more details.
+
+## Prerequisites:
+
+Ensure that you have the following tools installed locally:
+
+1. [aws cli](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html)
+2. [kubectl](https://Kubernetes.io/docs/tasks/tools/)
+3. [terraform](https://learn.hashicorp.com/tutorials/terraform/install-cli)
+4. [helm](https://helm.sh/docs/helm/helm_install/)
+
+## Deploy
```

Comment:
Please see the other pattern readmes for the "standard" README structure

Reply:
Done